cfme/test_framework/appliance_police.py

```diff
@@ -28,36 +28,39 @@ def appliance_police():
     if not store.slave_manager:
         return
     try:
-        port_numbers = {
-            'ssh': ports.SSH,
-            'https': store.current_appliance.ui_port,
-            'postgres': ports.DB}
-        port_results = {pn: net_check(pp, force=True) for pn, pp in port_numbers.items()}
+        appliance = store.current_appliance
+        available_ports = {
```

Comment:
**note** this code looks a bit hard to read to me, but off hand there doesn't seem to be a better way to do it